csharp/src/Google.Protobuf/WellKnownTypes/FieldMaskPartial.cs

```diff
@@ -59,7 +59,12 @@ internal static string ToJson(IList<string> paths, bool diagnosticOnly)
             if (firstInvalid == null)
             {
                 var writer = new StringWriter();
+#if DOTNET35
+                var query = paths.Select(JsonFormatter.ToCamelCase);
+                JsonFormatter.WriteString(writer, string.Join(",", query.Cast<string>().ToArray()));
```

Comment:
You shouldn't need the `Cast` call here, as `JsonFormatter.ToCamelCase` returns `string` anyway.